README.md

```diff
@@ -1 +1,57 @@
-# SHAPES
+# Console Shape Application Project
```

Comment:
Update the readme.md such that it should be attractive and readable so that one can easily understand the project by reading readme file.